src/components/Map/TileChartMap/TileChartMap.scss

```diff
@@ -272,6 +271,7 @@ x/y positioning on the SVG elements.
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
+
```

Comment:
We dropped IE11 support a long time ago.  Can we delete this rule?

Reply:
this  has been updrated